Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Login & Signup Flow/NonprofitSignupViewController.m

```diff
@@ -138,6 +104,7 @@ - (void)saveNonprofitDataToNonprofitObject {
 
 - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     if ([segue.identifier isEqualToString:@"unwindToUserSignup"]) {
+    } else if ([segue.identifier isEqualToString:@"loginSegue"]) {
```

Comment:
In this case I think code would be easier to read if you add a specific condition for the last `else` case instead of adding an empty `else if`.